### PR TITLE
DOC-8678: Feedback: Logging Overview - Document the KV_DISTRIBUTION channel

### PR DESCRIPTION
**Pull request by florence-crl**
Addresses DOC-8678

(1) Updated Logging Overview page, added KV_DISTRIBUTION channel and cockroach-kv-distribution.log destination. (2) Updated Configure Logs page, added kv-distribution file-group to default configuration.

### CODE REVIEW

File: src/current/v23.1/logging-overview.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ This allows you to group channels that log related information (e.g., operationa
...
 | [`SQL_PERF`]({% link {{ page.version.version }}/logging.md %}#sql_perf)                 | SQL executions that impact performance, such as slow queries (when enabled via the `sql.log.slow_query.latency_threshold` and/or `sql.log.slow_query.experimental_full_table_scans.enabled` [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %})).                                                                                    |
 | [`TELEMETRY`]({% link {{ page.version.version }}/logging.md %}#telemetry)               |  Telemetry events for internal usage.                                                                                                                                                                                                                                        |
+| [`KV_DISTRIBUTION`]({% link {{ page.version.version }}/logging.md %}#kv_distribution)   | Data distribution events, such as moving replicas between stores in a cluster, adding replicas to ranges, and removing replicas from ranges. |
```
Comment (rmloveland) [inline src/current/v23.1/logging-overview.md:43]: Suggest adding the following links to first use of each term:

- 'replicas': https://www.cockroachlabs.com/docs/v23.1/architecture/overview#architecture-replica 
- 'range': https://www.cockroachlabs.com/docs/v23.1/architecture/overview#range
- 'stores': https://www.cockroachlabs.com/docs/v23.1/cockroach-start#store

File: src/current/v23.2/logging-overview.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ This allows you to group channels that log related information (e.g., operationa
...
 | [`SQL_PERF`]({% link {{ page.version.version }}/logging.md %}#sql_perf)                 | SQL executions that impact performance, such as slow queries (when enabled via the `sql.log.slow_query.latency_threshold` and/or `sql.log.slow_query.experimental_full_table_scans.enabled` [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %})).                                                                                    |
 | [`TELEMETRY`]({% link {{ page.version.version }}/logging.md %}#telemetry)               |  Telemetry events for internal usage.                                                                                                                                                                                                                                        |
+| [`KV_DISTRIBUTION`]({% link {{ page.version.version }}/logging.md %}#kv_distribution)   | Data distribution events, such as moving replicas between stores in a cluster, adding replicas to ranges, and removing replicas from ranges. |
```
Comment (rmloveland) [inline src/current/v23.2/logging-overview.md:43]: Suggest adding the following links to first use of each term:

- 'replicas': https://www.cockroachlabs.com/docs/v23.1/architecture/overview#architecture-replica
- 'range': https://www.cockroachlabs.com/docs/v23.1/architecture/overview#range
- 'stores': https://www.cockroachlabs.com/docs/v23.1/cockroach-start#store

**Review by rmloveland — APPROVED** [review]
LGTM, had a suggestion to add some links

File: src/current/v23.2/logging-overview.md (hunk 2 above)
Comment (florence-crl) [reply src/current/v23.2/logging-overview.md]: updated

File: src/current/v23.1/logging-overview.md (hunk 1 above)
Comment (florence-crl) [reply src/current/v23.1/logging-overview.md]: updated

**Pull request merged**